ccd-definition/CaseEvent.json

```diff
@@ -145,7 +145,7 @@
     "LiveFrom": 42748,
     "CaseTypeID": "Shared_Storage_DRAFTType",
     "ID": "attendingHearing",
-    "Name": "Attending the court",
+    "Name": "Attending the hearing",
```

Comment:
That changes event name in dropdown. Don't we need to make similar change in tab configuration? At the moment tab still displays 'Attending the court' section.

Reply:
Fair point. Also some minor inconsistencies i.e 'Orders and directions needed' in dropdown but 'Orders and Directions' in tab (casing inconsistency). Going to grab some of Olivia's time to finalise.

Reply:
Caught up with Oliva. Content in tabs to be consistent with dropdown content. The solicitor table in 'people in the case' should also be nested within the applicant table to reflect 'Applicant' page. Out of scope for this story. 

![screen shot 2018-11-05 at 10 55 27](https://user-images.githubusercontent.com/4429628/47994144-56057480-e0e9-11e8-86dc-80205e7f7b37.png)